src/utils/column-helper.ts

```diff
@@ -52,6 +56,10 @@ export function setColumnDefaults(columns: TableColumn[]) {
   }
 }
 
+export function isNullOrUndefined<T>(value: T | null | undefined): value is null | undefined {
```

Comment:
Making a function here seems more readable. In the future, this can be moved to a more general purpose area.